tests/baselines/reference/asyncFunctionsAndStrictNullChecks.types

```diff
@@ -18,8 +18,8 @@ declare namespace Windows.Foundation {
 >progress : any
 
         then<U>(success?: (value: TResult) => IPromise<U>, error?: (error: any) => U, progress?: (progress: any) => void): IPromise<U>;
->then : { <U_1>(success?: (value: TResult) => IPromise<U_1>, error?: (error: any) => IPromise<U_1>, progress?: (progress: any) => void): IPromise<U_1>; <U>(success?: (value: TResult) => IPromise<U>, error?: (error: any) => U, progress?: (progress: any) => void): IPromise<U>; <U_2>(success?: (value: TResult) => U_2, error?: (error: any) => IPromise<U_2>, progress?: (progress: any) => void): IPromise<U_2>; <U_3>(success?: (value: TResult) => U_3, error?: (error: any) => U_3, progress?: (progress: any) => void): IPromise<U_3>; }
->     : ^^^   ^^       ^^^                                 ^^     ^^^                             ^^        ^^^                       ^^^^^^^^^^^^^^^^^^^ ^^       ^^^                               ^^     ^^^                 ^^        ^^^                       ^^^           ^^^   ^^       ^^^                       ^^     ^^^                             ^^        ^^^                       ^^^^^^^^^^^^^^^^^^^   ^^       ^^^                       ^^     ^^^                   ^^        ^^^                       ^^^^^^^^^^^^^^^^^^^
+>then : { <U_1>(success?: ((value: TResult) => IPromise<U_1>) | undefined, error?: ((error: any) => IPromise<U_1>) | undefined, progress?: (progress: any) => void): IPromise<U_1>; <U>(success?: (value: TResult) => IPromise<U>, error?: (error: any) => U, progress?: (progress: any) => void): IPromise<U>; <U_1>(success?: ((value: TResult) => U_1) | undefined, error?: ((error: any) => IPromise<U_1>) | undefined, progress?: (progress: any) => void): IPromise<U_1>; <U_1>(success?: ((value: TResult) => U_1) | undefined, error?: ((error: any) => U_1) | undefined, progress?: (progress: any) => void): IPromise<U_1>; }

```

Comment:
This also looks like a negative, given the original code did not have `| undefined`, right?